packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -177,6 +181,11 @@ const ComponentOverviewContent = ({ entity }: { entity: Entity }) => (
         <PagerDutyCard entity={entity} />
       </Grid>
     )}
+    {isSplunkOnCallAvailable(entity) && (
+      <Grid item md={6}>
+        <SplunkOnCallCard entity={entity} />
+      </Grid>
+    )}
```

Comment:
Again, would suggest we may not want to change this globally for the example app, but if needed adding it to the README for the plugin setup instructions?